### PR TITLE
Fix mechanumetc links

### PR DESCRIPTION
**Pull request by hamishwillee**
Fixes up linking errors arising from https://github.com/PX4/PX4-Autopilot/pull/24036

FYI @chfriedrich98 - note, I had to guess at one part. I'm going to merge, but would appreciate you checking the highlighted section.

### CODE REVIEW

File: en/config_rover/mecanum.md [hunk 1]
```diff
@@ -69,7 +68,6 @@ To set up [Acro mode](../flight_modes_rover/mecanum.md#acro-mode) navigate to [P
    5. Move the right-stick of your controller all the way to the left or right and observe the behaviour of the rover.
       Keep reducing the value of [RM_MAX_YAW_RATE](#RM_MAX_YAW_RATE) until you are satisfied with the maximum turn rate.
 
-   Remember to reset [RM_MAN_YAW_SCALE](#RM_MAN_YAW_SCALE) back to its original value.
```
Comment (hamishwillee) [inline en/config_rover/mecanum.md:72]: @chfriedrich98 For `RM_MAN_YAW_SCALE` I have just assumed that you don't need to reset the new values to anything - ie. RM_MAX_YAW_RATE

Can you sanity check the changes where RM_MAX_YAW_RATE has been removed?

**Comment by chfriedrich98** [comment]
This changes look good to me.
I will go through the docs again and scan for any features that are not documented yet to update everything to the latest state.

File: en/config_rover/mecanum.md (hunk 1 above)
Comment (chfriedrich98) [reply en/config_rover/mecanum.md:72]: LGTM

**Comment by hamishwillee** [comment]
Thanks. Saw you did that - lots of good fixes :-)

**Pull request merged**